styles.css

```diff
@@ -99,12 +101,13 @@ nav {
 .noqs {
```

Comment:
- [ ] The background for the header section is not aligned properly, kindly make sure the background is centered and the padding and margins on the headline section are similar to the Figma design this will make your project more accurate and shade light on your ability to create the design exactly as the client provides it

|Your design|Figma design|
|-|-|
|![Screenshot_20220901_213604](https://user-images.githubusercontent.com/25227022/188008399-c9ba26ca-d125-4ff8-bf8d-7cffe47cf630.png)|![Screenshot_20220901_213643](https://user-images.githubusercontent.com/25227022/188008425-5173fc0b-676f-4e7b-95ec-efbc1a8bc3be.png)|

Reply:
Kindly, I recommend you to follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉